app.js

```diff
@@ -3,24 +3,40 @@ const app = express();
 
 const { 
     getMessage,
-    getCategory
-} = require("./controllers/boardgames-controllers");
+    getCategory, 
+    } = require("./controllers/category-controllers");
+
+const {getReview} = require("./controllers/review-controller")
 
 //app.use(express.json()); ???
 
 app.get("/api", getMessage); // a connection to make sure everything is working as it should.
 app.get("/api/categories", getCategory); // a connection that returns an array of category objects.
-
+app.get("/api/reviews/:review_id", getReview); // a connection that returns an object with information about a review of a particular boardgame. 
 
 
 
 //Error handling suite 
 
+//Error handling for status 400
+app.use((error, request, response, next) => {
+    if (error.code ==="22P02") {
+        response.status(400).send({ msg: "bad request"});
+    } else {
+        next(error);
+    }
+})
+
 //Error handling for status 404
 app.use("/*", (request, response, next) => {
-    response.status(404).send({ msg: "Route not found" });
+    response.status(404).send({ msg: "page not found" });
 });
 
+//Error handling for unknown error messages
+app.use((error, request, response, next) => {
+    response.status(error.status).send( {msg: error.msg });
+    console.log({msg: error.msg}, "<---Error Message")
```

Comment:
Remove console.logs before merging. If your end point is finished and ready to merge with main, we should not need these for troubleshooting

Reply:
I thought this after I pushed the branch... at least you can see I am having lots of trouble and then getting myself out of it ;)